pom.xml

```diff
@@ -36,7 +36,7 @@
     <version>2.13-SNAPSHOT</version>
     <packaging>pom</packaging>
     <name>Pipeline Stage View Plugin: Parent POM</name>
-    <url>https://wiki.jenkins.io/display/JENKINS/Pipeline+Stage+View+Plugin</url>
+    <url>https://github.com/jenkinsci/pipeline-stage-view-plugin</url>
```

Comment:
I can't comment in the right places, but I think you should also switch the URLs for the two modules to the GitHub repo URL, since those are the actual plugins that will be shown on https://plugins.jenkins.io/.